src/test/java/core/basesyntax/service/WriterServiceTest.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.model.Transaction;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.Test;
+
+public class WriterServiceTest {
```

Comment:
the same as for reader service. All tests are useless. You don't test the behavior of the service at all. Try to start this test with coverage, and you see that WriterServiceImpl class has 0% coverage.

Reply:
Not fixed, 
you need to call the methods of this service"WriterServiceImpl" and not write the same logic that is in it